content-src/asrouter/templates/Trailhead/Trailhead.jsx

```diff
@@ -131,9 +132,33 @@ export class _Trailhead extends React.PureComponent {
     return str.value;
   }
 
+  onCardAction(action) {
+    let actionUpdates = {};
+
+    if (action.type === "OPEN_URL") {
+      let url = new URL(action.data.args);
+      let params = new URLSearchParams(url.search.slice(1));
```

Comment:
Do we need to create a new `Params` vs just directly using `url.searchParams` ? If we reuse, appending will result in `url` getting updated too.

Reply:
Doh. MDN says `searchParams` is read only: https://developer.mozilla.org/en-US/docs/Web/API/URL. But a quick test in the console confirms that works.